src/twisted/newsfragments/9358.bugfix

```diff
@@ -0,0 +1 @@
+twisted.internet.iocpreactor.send no longer attaches send buffer to completion event, saving memory.
```

Comment:
Is `twisted.internet.iocpreactor.send` a real thing :)
twisted.internet.iocpreactor is a package and I don't see any send there
https://github.com/twisted/twisted/blob/twisted-17.9.0/src/twisted/internet/iocpreactor/__init__.py

Reply:
I need to update the newsfragment.  It should be twisted.internet.iocpreactor.iocpsupport.send.